### ISSUE TITLE
Doesn't seem to work with Tiny MCE 6

### ISSUE THREAD

**Issue by DedMaxim** [Paradox81ru/TinyMCE_v5_spoiler#1]
Doesn't seem to work with Tiny MCE 6. Tiny spits out an error on plugin load

**Comment by Paradox81ru** [comment]
I haven't tried version 6 yet, of course, I still have version 5 installed. But purely visually, the API doesn't seem to have changed. It should work. Check the plugin installation.

**Comment by DedMaxim** [comment]
I have downloaded the plugin, extracted "spoiler" folder out of it and pasted to "tiny/plugins/".
Then I added all necessary info into the script:
	<script>
		tinymce.init({
			selector: '#description',
			plugins: 'autolink link media image table charmap wordcount quickbars insertdatetime spoiler',
			toolbar: 'undo redo | styles | fontsize | alignleft aligncenter alignright alignjustify | bold italic underline | link image media | spoiler',
			menu: { insert: {items: 'spoiler'}},
			content_css: '/tiny/tiny_style.css',
			skin: 'tinymce-5-dark',
			quickbars_insert_toolbar: false,
			add_form_submit_trigger : true,
			language: '<?php echo $tiny_lang ?>',
			setup: function(editor)
			{
				editor.on('paste', function(event)
				{
					var numChars = 0;
					setTimeout(function()
					{
						numChars = tinymce.activeEditor.plugins.wordcount.body.getCharacterCount();
						document.getElementById("description_chars").innerHTML = numChars + "/40000";
						if (numChars > 40000)
						{
							alert('<?php echo $lang['description_limit'] ?>');
						}
					}, 0);
				});


				editor.on('input', function(event)
				{
					var numChars = tinymce.activeEditor.plugins.wordcount.body.getCharacterCount();
					document.getElementById("description_chars").innerHTML = numChars + "/40000";
					if (numChars > 40000)
					{
						alert('<?php echo $lang['description_limit'] ?>');
					}
				});
			}
		});
	</script>
	
However, I get an error on script load.
	
![image](https://user-images.githubusercontent.com/41780875/213248795-c0675707-2976-4c54-a99b-c9d915ac51f7.png)


Do I need to somehow include "init_tinymce.js" file from the archive as well or do something with it?

**Comment by Paradox81ru** [comment]
No, the file is "_init_tinymce.js_" initializes tinymce, you already did it in your script "_tinymce.init({...})_"
I have tried to upgrade tinymce to version 6 and started getting the same error. After all, 6 tinymce is not fully compatible with 5. Even the some native plugins of version 5 on 6 does not work. I'll try to figure out what has changed.

**Comment by Paradox81ru** [comment]
I've corrected something here, there seems to be no mistake. Since it turns out that the plugin for version 5 of tinymce and 6 are different, then it is necessary to publish the corrected plugin in a new branch. And now, unfortunately, I don't have time to finally check everything and issue a new branch. Therefore, I can only offer to download the corrected plugin from Yandex.disk at the link: https://disk.yandex.ru/d/QXz-XG4Y5cgXmw.

**Comment by DedMaxim** [comment]
Awesome. Thanks a lot!
